### DISCUSSION TITLE
FNIRSI GC-01: Rad Pro doesn't start up

### DISCUSSION THREAD

**Discussion by yanakis71** [Gissio/radpro#146] [General]
hi.

Everything worked fine with 2.0 but I haven't used it for 3-4 months. Now I tried to turn it on and it does nothing. Dead!

What to do now?

Apologies if it was already answered.

Many thanks!

**Comment by yanakis71** [comment]
I manage to power it on by inserting a needle into the right hole of USB port and holding power button on. problem is after I turn it off, it wont go back unless I insert the needle again. Also, it loses date an time.

**Reply by jonwhite227** [reply]
It sounds like the back up battery (coin cell mounted on the circuit board) might need checking / replacing.

**Reply by yanakis71** [reply]
Indeed, that small watch like battery has now 0.3 V. Ordered a couple. Thanks

**Reply by jonwhite227** [reply]
Glad to help 👍
